.github/workflows/nextpnr.yml

```diff
@@ -43,16 +44,24 @@ jobs:
 
     - run: echo "$(pwd)/.github/bin" >> $GITHUB_PATH
 
-    - run: dockerBuild nextpnr:ice40      nextpnr ice40
-    - run: dockerBuild nextpnr:icestorm   nextpnr icestorm
-    - run: dockerBuild nextpnr:ecp5       nextpnr ecp5
-    - run: dockerBuild nextpnr:prjtrellis nextpnr prjtrellis
-    - run: dockerBuild nextpnr            nextpnr
+    - run: dockerBuild nextpnr:ice40       nextpnr ice40
+    - run: dockerBuild nextpnr:icestorm    nextpnr icestorm
+    - run: dockerBuild pkg:nextpnr-ice40   nextpnr pkg-ice40
```

Comment:
Please, move `pkg` images first:

- pkg:nextpnr-ice40
- nextpnr:ice40
- nextpnr:icestorm
- pkg:nextpnr-ecp5
- ...

From smallest to largest. 

I would try keeping the same order for testing and pushing.